code/game/objects/items/weapons/shields.dm

```diff
@@ -624,3 +624,50 @@
 		set_light(1.5, 1.5, COLOR_LIGHTING_RED_BRIGHT)
 	else
 		set_light(0)
+		
+/obj/item/shield/parrying
+	name = "parrying dagger"
+	desc = "A parrying dagger forged in a specific way to easily parry other melee attacks with ease, sacrificing its lethality in exchange."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "dagger"
+	item_state = "dagger"
+	flags = CONDUCT
+	w_class = ITEM_SIZE_SMALL
+	slot_flags = SLOT_BELT
+	force = WEAPON_FORCE_NORMAL
+	throwforce = WEAPON_FORCE_DANGEROUS
+	throw_speed = 2
+	throw_range = 6
+	armor_list = list(melee = 15, bullet = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	matter = list(MATERIAL_PLASTEEL = 5, MATERIAL_PLASTIC = 2)
+	price_tag = 250
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	max_durability = 100 //So we can brake and need healing time to time
+	durability = 125
+	var/cooldown = 0 //shield bash cooldown. based on world.time
+	var/picked_by_human = FALSE
+	var/mob/living/carbon/human/picking_human
+
+/obj/item/shield/parrying/handle_shield(mob/user)
+	. = ..()
+	if(.) playsound(user.loc, 'sound/weapons/Genhit.ogg', 50, 1)
```

Comment:
```suggestion
	if(.) playsound(user.loc, 'sound/weapons/shield/shieldmelee.ogg', 50, 1)
```
Could pass it out as the sound of clashing steel, or add another parry sound file!